README.md

```diff
@@ -325,7 +325,7 @@ NVM_IOJS_ORG_MIRROR=https://iojs.org/dist nvm install iojs-v1.0.3
 ### .nvmrc
 
 You can create a `.nvmrc` file containing version number in the project root directory (or any parent directory).
-`nvm use`, `nvm install`, `nvm exec`, `nvm run`, and `nvm which` will all respect an `.nvmrc` file when a version is not supplied.
+`nvm use`, `nvm install`, `nvm exec`, `nvm run`, and `nvm which` will all respect an `.nvmrc` file when a version is not supplied in the command line.
```

Comment:
LGTM - let's change this to *on* the command line :-)

Reply:
Hmm... I was thinking about that too. but it did sound strange, but English is not my first language, soo... I might be wrong, and for sure are, but still... are we sure that `on` is the best option here? :) I mean where are we when we type in the CLI, in it or on it :D

Reply:
I think both are correct, but the more common expression I've seen is "on" the command line.